src/style.css

```
@@ -1 +1,59 @@
-/* Styles go here */
+* {
+	font-family: 'Robot';
+	font-size: 12px;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
```

reset margin and padding  for tags it is necessary
```suggestion
	box-sizing: border-box;
}
```